modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,70 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    is_a: Template
+    description: >
+      A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      consortiumName:
+        range: string
+        required: true
+      themeName:
+        range: string
+        required: true
+      doi:
+        range: string
+        required: true
+      journal:
+        range: string
+        required: true
+      pmid:
+        range: integer
+        required: true
+      pubmedUrl:
+        range: string
+        required: true
+      title:
+        range: string
+        required: true
+      year:
+        range: integer
+        required: true
+      diseaseFocus:
+        range: string
+        multivalued: true
+        required: false
+      authors:
+        range: string
+        multivalued: true
+        required: false
+      abstract:
+        range: string
+        required: true
+      assay:
+        range: string
+        multivalued: true
+        required: false
+      accessibility:
+        range: string
+        required: true
+
+  NFPublication:
+    is_a: PublicationTemplate
+    description: A slightly more specialized publication concept intended for the specific scope of the NF Portal.
+    attributes:
+      studyUse:
+        range: string
+        required: true
+      nfType:
```

Comment:
What is `nfType`?

Reply:
I was thinking this would be the disease type of nf (Type 1, 2, schwannomatosis)

Reply:
Replaced with diseasefocus